banner_react/src/Banner.js

```diff
@@ -40,11 +40,9 @@ class Banner extends Component {
         if (this.props.transition) {
             if (this.state.bannerStat === 0 || this.state.bannerStat === 2) {
                 this.setState({ bannerStat: this.state.bannerStat + 1 });
-                this.myTimer = setInterval(() => {
-                    this.props.whenTransition()
-                }, this.state.transTime / 30);
+                this.transitioning();
                 setTimeout(() => {
-                    this.clearTimer(this.myTimer);
+                    this.clearTimer(this.myTimer._id);
```

Comment:
I don't think this is necessary? 🤔

Reply:
i'm being honest when i said i have no idea what i am doing